Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
Maybe I should wait till separable menu view is merged back to master? 
I can add the multiline menu support later. It works for my project in my own repo for now. If you think this feature would be good, I'm happy to rework on the multiline text thing with the new code of separable menu view later.